### ISSUE TITLE
Setting to set default subtitle behavior on a device.

### ISSUE THREAD

**Issue by Trevo525** [jellyfin/jellyfin-androidtv#3705]
### Describe the feature you'd like

I would love an option to change a setting on the client that makes it so that it automatically turns on subtitles. We turn on subtitles on everything we watch and it would be nice to have an option that just automatically does this for everything you play on this device. 

I imagine a dropdown box that says something like `Automatically use subtitles` with three options 
- "Off"
- "Select the first subtitle (if it exists)"
- "Select the first option in this devices language"

Thanks!

**Comment by Pentaphon** [comment]
Agreed with this. Many people get annoyed by the lack of this setting. We could set it on the server but perhaps an override would be useful.

**Comment by Organizer21** [comment]
Agreed, sounds like a great option. On a related note; it seems there's no option to to change the font and or size on subtitles the fly (fire tv app)? I understand some sort of scaling is being used, but in my case the font are way too large, we really need options to at least control subtitle font size.
